src/components/topVehicleSharedBadge/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Box, useTheme } from '@chakra-ui/react';
+
+import { Brand } from 'src/types/brand';
+
+import AspectRatio from '../aspectRatio';
+
+type Props = {
+  aspectRatio?: number;
+};
+
+const badgeText = {
```

Comment:
I mulled over this a bit. An alternative would be to accept the text and let the project control it. But that would result in an unnecessary duplication so I went with brand dependent text

Reply:
I wonder if there might be a requirement to have that per language?

Reply:
I don't think so, but if there is we do have some translations in the components package so this would fit right in